framework/console/controllers/MigrateController.php

```diff
@@ -273,6 +273,35 @@ protected function addMigrationHistory($version)
 
     /**
      * @inheritdoc
```

Comment:
there is no parent method to inherit docs from. `refreshDatabase` method should be implemented in BaseMigrateController and print something like `'This command is not implemented in ' . get_class($this)`